src/scripts/OSFramework/OSUI/Event/DOMEvents/Listeners/ListenerEnum.ts

```diff
@@ -10,5 +10,6 @@ namespace OSFramework.OSUI.Event.DOMEvents.Listeners {
 		OrientationChange = 'window.onorientationchange',
 		ScreenOnScroll = 'screen.onscroll',
 		WindowResize = 'window.onresize',
+		WindowMessage = 'window.message',
```

Comment:
@rugoncalves Some of these things, if they have no impact on O11 should we also remove them? Or are we doing overengineering? I already discussed this with @joselrio and I don't have any clear decision about how far should we go with. When is a file or something impacting performance or functionality, sure. otherwise not sure. Thoughts?

Reply:
Same as before.